retinanet/infer.py

```diff
@@ -138,5 +135,26 @@ def infer(model, path, detections_file, resize, max_size, batch_size, mixed_prec
                     coco_eval.evaluate()
                     coco_eval.accumulate()
                 coco_eval.summarize()
+                results = coco_eval.stats
+                # Create TensorBoard writer
+                if logdir is not None:
+                    from tensorboardX import SummaryWriter
+                    if is_master and verbose:
+                        print('Infer writer: Writing TensorBoard logs to: {}'.format(logdir))
+                    writer = SummaryWriter(logdir=logdir)
+                    if results != []:
+                        writer.add_scalar('Average Precision (AP) IoU=0.50:0.95  area all  maxDets=100', results[0],iteration)
```

Comment:
It would be better to use hierarchical tags here again. So "Average Precision/IoU=/areaY/maxDets100" or something like that